bundles/paikkatietoikkuna/inspire/Flyout.js

```diff
@@ -33,10 +33,7 @@ Oskari.clazz.define('Oskari.inspire.Flyout',
                 return;
             }
 
-            ReactDOM.render(
-                <FlyoutContent textContent={ this.flyoutContent } />,
-                root
-            );
+            ReactDOM.render(<Message bundleKey="flyoutContentMessage" messageKey={ this.flyoutContent } allowHTML={true} />, root);
```

Comment:
Try this one: 
```suggestion
            ReactDOM.render(<Message bundleKey={ this.instance.getName() } messageKey="flyoutContent.content" allowHTML={true} />, root);
```

Reply:
Also 
```
const root = document.getElementsByClassName('oskari-flyoutcontent inspire')[0];
```
could probably be changed to:
```
const root = this.container[0];
```

Reply:
Then you could remove this.flyoutContent and this.loc = Oskari.getMsg.bind(null, 'inspire'); lines from constructor